README.dev.md

```diff
@@ -243,7 +272,8 @@ npm install
 ./tools/download-zxcvbn
 ./tools/emoji_dump/build_emoji
 ./scripts/setup/generate_secrets.py -d
-sudo cp ./puppet/zulip/files/postgresql/zulip_english.stop /usr/share/postgresql/9.3/tsearch_data/
+sudo cp ./puppet/zulip/files/postgresql/zulip_english.stop /usr/share/postgresql/9.3/tsearch_data/ (Linuxes)
+sudo cp ./puppet/zulip/files/postgresql/zulip_english.stop /var/postgresql/tsearch_data/ (OpenBSD)
```

Comment:
People usually just copy-paste this whole block, and the parens will probably make that fail.  Maybe it's OK to add a little if statement for whether it's OpenBSD or other (Mac should be the same path as Linux I think).